src/main/java/com/xiaomi/infra/pegasus/rpc/Cluster.java

```diff
@@ -55,13 +58,19 @@ public static Cluster createCluster(Properties config) throws IllegalArgumentExc
         Integer.parseInt(
             config.getProperty(
                 PEGASUS_PUSH_COUNTER_INTERVAL_SECS_KEY, PEGASUS_PUSH_COUNTER_INTERVAL_SECS_DEF));
+
+    int maxAllowedWriteSize =
+        Integer.parseInt(
+            config.getProperty(PEGASUS_MAX_ALLOWED_WRITE_SIZE, PEGASUS_MAX_ALLOWED_WRITE_SIZE_DEF));
```

Comment:
Should we provide a config to let user set the max allowed write size?

Reply:
if no config, it will use default value: 1MB